src/services/infra/InfraService.ts

```diff
@@ -190,7 +183,7 @@ export default class InfraService {
     repoName: string,
     primaryDomain: string,
     subDomainSettings: SubDomainSettings
-  ): Promise<Err<never, unknown> | Ok<SiteLaunchCreateParams, never>> => {
+  ): Promise<Err<never, AmplifyError> | Ok<SiteLaunchCreateParams, never>> => {
```

Comment:
have you tried typing this as `Promise<Result<SiteLaunchCreateParams, AmplifyError>>` (and next time `ResultAsync<SiteLaunchCreateParams, AmplifyError>`)?

Reply:
Sure: d1e58db9